src/ucp/wireup/select.c

```diff
@@ -896,7 +906,8 @@ static ucs_status_t ucp_wireup_add_am_bw_lanes(ucp_ep_h ep, const ucp_ep_params_
         }
     }
 
-    return found ? UCS_OK : status;
+    return (found || (ep->worker->context->config.ext.max_eager_lanes < 2)) ?
```

Comment:
why should we return error if not found? AFAIU am_bw_lane[0] always exists.. so should always return OK
then, can rename the variable "found" to "num_lanes" and start with "num_lanes=1"

Reply:
done